pkg/clouduploader/conn.go

```diff
@@ -250,3 +272,20 @@ func (c *Conn) Destroy(rw ReadWriter, opType ServerOperation) {
 		return
 	}
 }
+
+func getPartSize(config map[string]string) (val int64, err error) {
```

Comment:
I think we should check if the provided size is less than `5Mi`, then throw an error or use the default size. 
What are your thoughts on this?

Reply:
If provided size is less than 5 Mi then uploading will throw an error.
We can use the default size or derive it from file in case of invalid size but that needs to be reported to user. So better to report an error or failed to backup. 
I will add explicit check for the provided size in plugin initialization, that will save the backup time by avoiding snapshot/resource creation.

Reply:
Thanks for the clarification @mynktl 

>I will add explicit check for the provided size in plugin initialization, that will save the backup time by avoiding snapshot/resource creation.

This will be good. :+1: